examples/CMakeLists.txt

```diff
@@ -28,7 +28,8 @@ endfunction(progress_example)
 progress_example(changecoords changecoords/changecoords.F90)
 progress_example(getdihedral getdihedral/getdihedral.F90)
 progress_example(gpscf_dist gpdist/gpscf_dist.F90)
-progress_example(gpmd_dist gpdist/gpmd_dist.F90)
+#MEW Comment out as doesn't build on crusher
```

Comment:
Do we want to turn off that one because it doesn't build on one platform? Why does it not build?